py/server/deephaven/csv.py

```diff
@@ -75,9 +75,9 @@ def read(
                 dht.byte: _JParsers.BYTE,
                 dht.char: _JParsers.CHAR,
                 dht.short: _JParsers.SHORT,
-                dht.int64: _JParsers.INT,
+                dht.int32: _JParsers.INT,
                 dht.long: _JParsers.LONG,
```

Comment:
Do int64 and float64 correctly map to long and double?

Reply:
yes, they are the same objects.